kuma/search/serializers.py

```diff
@@ -6,13 +6,20 @@
 
 
 class SearchQuerySerializer(serializers.Serializer):
-    q = serializers.CharField(required=False)
+    q = serializers.CharField(required=False, max_length=1024)
```

Comment:
Where does this come from? Why 1024? Why not 123? or 123456?

Reply:
This number is now repeated in 3 places. Can you please move it to `settings/common.py` and here do:
```python
q = serializers.CharField(required=False, max_length=settings.SEARCH_Q_MAXLENGTH)
```
(might need some linebreaks and indentation)

Then in `settings/common.py` you can write *why* you chose 1024. E.g.

```python
# The default 1024 is because ........
SEARCH_Q_MAXLENGTH = config('SEARCH_Q_MAXLENGTH', default=1024, cast=int)
```